### PR TITLE
add esp32c3 example

### PR DESCRIPTION
**Pull request by kons-9**
Hi! I made esp32c3 example!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,3 +85,15 @@ cargo build --release --example draw_ferris
 | DC         | gpio13                |
 | CS         | GND                   |
 | LED        | gpio12                |
+
+### M5Stamp C3U
+| ST7735R Pin | Raspberry Pi Pico Pin |
```
Comment (sajattack) [inline README.md]: you forgot to update the name here

File: esp32c3-examples/examples/draw_ferris.rs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    let peripherals = Peripherals::take().unwrap();
+
+    let spi = peripherals.spi2;
+    let sclk = peripherals.pins.gpio8;
+    let sdo = peripherals.pins.gpio10;
+    let sdi = Option::<Gpio0>::None;
+    let cs = Option::<Gpio0>::None;
+    let driver_config = Default::default();
+    let hertz = Hertz(30_000_000);
```
Comment (sajattack) [inline esp32c3-examples/examples/draw_ferris.rs]: this might be a bit clearer if you use MegaHertz(30)

**Review by sajattack — CHANGES_REQUESTED** [review]
as above

**Comment by kons-9** [comment]
Thank you for reviewing!
I fixed them.

**Review by sajattack — APPROVED** [review]

**Comment by sajattack** [comment]
Thanks for contributing!

**Pull request merged**